cspell.json

```diff
@@ -64,19 +64,32 @@
   "words": [
     "apikey",
     "buildhome",
+    "bodycol",
     "callout",
     "checkmark",
+    "closebutton",
```

Comment:
I think if we rename `closebuttonProps` to `closeButtonProps`, this will be resolved.